fuse_models/include/fuse_models/unicycle_2d_state_cost_functor.h

```diff
@@ -147,27 +145,36 @@ bool Unicycle2DStateCostFunctor::operator()(
   const T* const acc_linear2,
   T* residual) const
 {
-  T position_pred[2];
```

Comment:
I believe the comment I made above applies here as well. Indeed, I suspect with your change the compiler must be complaining of unused arguments, isn't it?